[ #3 ]  Add Node.js code samples for each feature described in the README (#5)
redislabs-training/hitc-new-in-redis-62: nodejs/README.md

# Examples using 'node-redis' client

This repository contains example data in nodejs using node-redis client.

## Setup and Start

### Sample Data

> Use the main readme.md for sample data load. Some examples are using preloaded keys.

Reviewer: ```suggestion
Use the main README.md for sample data load. Some examples are using preloaded keys.
```

```bash
docker exec -it hitc_redis sh /sampledata/load_all.sh
```

### Reload Sample Data

> Some tests are modifying the Sample Data so the next run will not represent the same dataset for tests. To reload everything use the next commands:

Reviewer: ```suggestion
Some tests modify the Sample Data so the next run will not start with the same dataset. To reload everything use the following commands:
```

   Mint Choc Chip
   Vanilla Bean
   86399
   43200
2. Strings: New Alternatives to the GET Command
   halftimeoff
   null
   freeicecream
   3600
   freeicecream
   7200
   freeicecream
   86399
   freeicecream
   43200
3. Streams: MINID Trimming Strategy
   [{"id":"1617307200000-0","message":{"couponcode":"weekendsale","userid":"6935"}}]
   [{"id":"1617314919000-0","message":{"couponcode":"weekendsale","userid":"5577"}}]
   250
   16
   234
   [{"id":"1617307696000-0","message":{"couponcode":"weekendsale","userid":"529"}}]
   1665019772629-0
   230
4. Hashes: HRANDFIELD Command
   {"10":"Szechuan Shredded Beef","11":"King Prawns with Cashew Nuts","12":"Roast Duck with Plum Sauce","13":"Grilled Lemon Chicken","14":"Chicken with Pineapple","15":"Beef with Broccoli","16":"Malaysian Chicken Curry","17":"House Special Chow Mein","18":"Mushroom Fried Rice","19":"Fried Mushrooms with Black Bean Sauce","20":"Singapore Fried Rice"}
   15
   {"14":"Chicken with Pineapple","16":"Malaysian Chicken Curry"}
   {"13":"Grilled Lemon Chicken","17":"House Special Chow Mein"}
5. Sets: SMISMEMBER Command
   [ '3', '5', '19', '27', '33', '56' ]
   [ false, true, false, true, false, true ]
```

> ``Note - results might be abit offdue to random functions used``

Reviewer: ```suggestion
``Note - results might be a bit off due to random functions used``
```
